client/my-sites/media-library/content.jsx

```diff
@@ -136,11 +143,19 @@ module.exports = React.createClass( {
 					);
 					break;
 				default:
-					message = this.translate(
-						'The file could not be uploaded because an error occurred while uploading.',
-						'%d files could not be uploaded because errors occurred while uploading.',
-						i18nOptions
-					);
+					if ( this.props.overMediaLimit ) {
+						message = this.translate(
+							'The file could not be uploaded because you have reached your plan storage limit.',
+							'%d files could not be uploaded because you have reached your plan storage limit.',
```

Comment:
If we include a count number in the plural version, we're also supposed to include it in the singular version because some languages use a singular case for numbers like 21, 31, etc. This was previously a linted issue, but we may have lost this check in some recent changes (I haven't confirmed, but /cc @aduth). https://github.com/Automattic/wp-calypso/pull/3849/files#diff-26639d57c22772d167c024b7f4b06fa6L257